Userland/Libraries/LibWeb/Internals/Internals.idl

```diff
@@ -36,4 +36,6 @@ interface Internals {
 
     undefined enableCookiesOnFileDomains();
     undefined expireCookiesWithTimeOffset(long long seconds);
+
+    DOMString get_computed_label(Element element);
```

Comment:
Every other API in this file uses camelCase rather than snake_case, so let's be consistent :)

Reply:
> Every other API in this file uses camelCase rather than snake_case, so let's be consistent :)

✅  Made it so